### PR TITLE
Fix behavior when tapping more detail button on EventMapView

### PR DESCRIPTION
**Pull request by shin-usu**
## Issue
- close #ISSUE_NUMBER

## Overview (Required)
- Fix behavior when tapping more detail button on EventMapView

## Links
- https://droidkaigi.slack.com/archives/C06LTBTL83U/p1724719868793069

## Movie (Optional)
Before | After
:--: | :--:
<video src="https://github.com/user-attachments/assets/aa54ae24-ac98-4169-92ee-eaae47088698" width="300" > | <video src="https://github.com/user-attachments/assets/a8ac27ef-3dcd-45a2-8af9-9de24eaaf1e8" width="300" >

### CODE REVIEW

File: app-ios/Sources/EventMapFeature/EventMapReducer.swift [hunk 1]
```diff
@@ -12,18 +14,21 @@ public struct EventMapReducer: Sendable {
...
         public var selectedFloorMap: FloorMap = .first
         public var events: [EventMapEvent] = []
+        public var url: IdentifiableURL?
         
         public init() { }
     }
     
-    public enum Action {
+    public enum Action: BindableAction {
+        case binding(BindingAction<State>)
```
Comment (charles-b-stb) [inline app-ios/Sources/EventMapFeature/EventMapReducer.swift:23]: Is this parameter unused?

.binding below appears to not need or use any parameters.

Comment (shin-usu) [reply app-ios/Sources/EventMapFeature/EventMapReducer.swift:23]: It is necessary code to bind reducer's state at TCA.
ref: https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/bindings

Comment (charles-b-stb) [reply app-ios/Sources/EventMapFeature/EventMapReducer.swift:23]: Understood. Comment withdrawn.

**Comment by shin-usu** [comment]
I forgot to add test case, so I added it.
https://github.com/DroidKaigi/conference-app-2024/pull/828/commits/5277a19de7537fe367e3c3bb27eb39ab43420d6f

**Review by MrSmart00 — APPROVED** [review]
Thanks!
LGTM👍

**Pull request merged**